apps/nextjs-website/react-components/components/common/Common.tsx

```diff
@@ -164,6 +165,14 @@ const StringBody = (
     variant={variant}
     style={{ color: textColor, marginBottom }}
     align={textAlign}
+    sx={{
+      '& a': {
```

Comment:
```suggestion
      'a': {
```
This needs to be tested, but I don't think the "&" here is needed since the simple `a` should be targeting all links children of Typography